dynamometer-workload/src/test/java/com/linkedin/dynamometer/workloadgenerator/TestWorkloadGenerator.java

```diff
@@ -105,5 +111,12 @@ private void testAuditWorkload() throws Exception {
     assertTrue(dfs.getFileStatus(new Path("/tmp/test1")).isFile());
     assertTrue(dfs.getFileStatus(new Path("/tmp/testDirRenamed")).isDirectory());
     assertFalse(dfs.exists(new Path("/denied")));
+
+    assertTrue(dfs.exists(new Path(auditOutputPath)));
+    FSDataInputStream auditOutput = dfs.open(new Path(auditOutputPath + "/part-r-00000"));
```

Comment:
Use try-with-resources to make sure this is closed.

Reply:
TIL, cool